.gitignore

```diff
@@ -16,3 +16,4 @@
 
 # Ignore the generated pact
 spec/pacts/publishing_api-content_store.json
+.DS_Store
```

Comment:
There's nothing wrong in adding it to the repo, it should be there to avoid the mistake of adding a file that shouldn't be in the repo in the first place. If someone doesn't use a OSX machine that will have no impact in their workflow whatsoever.